src/core/lombok/Version.java

```diff
@@ -0,0 +1,26 @@
+package lombok;
+
+/**
+ * @See VersionableUtils
+ * @author Al3x G0m3z
+ *
+ */
+public enum Version
+{
+    v1,
+    v2,
+    v3,
+    v4,
+    v5,
+    v6,
+    v7,
+    v8,
+    v9,
+    v10,
+    v11,
+    v12,
+    v13,
+    v14,
+    v15,
+    v16
+}

```

Comment:
Not sure if this is the most elegant solution to this problem. Using an enum like this restricts the user to only 16 versions... why don't you just have the annotation accept an `int`?

Reply:
@lexfaraday The README does still involve the enum.